app/res/layout-land/mainnew.xml

```diff
@@ -28,69 +28,73 @@
                 android:scaleType="fitCenter"
                 android:src="@drawable/ccbanner" />
 
-            <LinearLayout
-                android:id="@+id/home.left.inner"
-                style="@style/MainSidePaneLeft"
-                android:layout_width="fill_parent"
-                android:layout_height="fill_parent"
-                android:layout_below="@id/main_top_banner"
-                android:layout_centerVertical="false"
-                android:layout_weight="1"
-                android:orientation="vertical" >
-
-                <Button
-                    android:id="@+id/home.start"
-                    android:layout_width="fill_parent"
-                    android:layout_height="wrap_content"
-                    android:drawableLeft="@drawable/pencil"
-                    android:text="Start" />
-
-
+            <ScrollView
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_below="@id/main_top_banner" >
                 <LinearLayout
-                    android:layout_width="match_parent"
-                    android:orientation="vertical" android:paddingTop="3dp" android:id="@+id/home.formspanel" android:layout_height="wrap_content">
-
-                    <TextView
-                        android:id="@+id/home.formrecords.label"
+                    android:id="@+id/home.left.inner"
+                    style="@style/MainSidePaneLeft"
+                    android:layout_width="fill_parent"
+                    android:layout_height="fill_parent"
+                    android:layout_below="@id/main_top_banner"
+                    android:layout_centerVertical="false"
+                    android:layout_weight="1"
+                    android:orientation="vertical" >
+
+                    <Button
+                        android:id="@+id/home.start"
                         android:layout_width="fill_parent"
                         android:layout_height="wrap_content"
-                        android:text="Form Records"
-                        android:textAppearance="?android:attr/textAppearanceMedium" android:gravity="center_horizontal"/>
+                        android:drawableLeft="@drawable/pencil"
+                        android:text="Start" />
+
 
                     <LinearLayout
-                        android:layout_width="fill_parent"
-                        android:layout_height="wrap_content"
-                        android:orientation="horizontal" >
+                        android:layout_width="match_parent"
+                        android:orientation="vertical" android:paddingTop="3dp" android:id="@+id/home.formspanel" android:layout_height="wrap_content">
 
-                        <Button
-                            android:id="@+id/home.forms.incomplete"

```

Comment:
Isn't this ID used in the form to accept the button click? Doesn't that id need to be re-linked?